controllers/glanceapi_controller.go

```diff
@@ -743,8 +748,9 @@ func (r *GlanceAPIReconciler) generateServiceConfig(
 	}
 
 	// Configure the internal GlanceAPI to provide image location data, and the
-	// external version to *not* provide it.
-	if instance.Spec.APIType == glancev1.APIInternal {
+	// external version to *not* provide it; if we don't split, the resulting
+	// GlanceAPI instance will provide it.
+	if instance.Spec.APIType == glancev1.APIInternal || instance.Spec.APIType == glancev1.APISingle {
```

Comment:
Just curious to know don't we have something similar to "in" function in python here like 'x' in [a,b,c,d] ?

Reply:
I don't think so, or at list this can be achieved via the following:

```golang
func checkAPIType(current string, myarray []string) bool {
    for _, b := range myarray.(string) {
        if b == current {
            return true
        }
    }
    return false
}
```
and then I can simply:

```golang
if checkAPIType(instance.Spec.APIType,  []string{string(glancev1.APIInternal), string(glancev1.APISingle)}) {
...
}
 ```
 
but it makes less obvious what we're trying to achieve in that logic, and instead of an additional `||` we're going to add more complexity building an array (on-the-fly) for the purpose of making an iteration.

Reply:
ack, make sense, I thought there could have been inbuilt method for this. May be if worth putting efforts, we can add such small methods later in common modules.

Reply:
+1 on further improvements like that one to improve readability

Reply:
nice, that's exactly what I had in mind to implement the equivalent of `"string" in array`! Noted for a follow up change.